client/app/scripts/components/cloud-link.js

```diff
@@ -57,7 +57,7 @@ class LinkWrapper extends React.Component {
   render() {
     const { url, children, ...props } = this.props;
     if (!url) {
-      return children;
+      return React.isValidElement(children) ? children : (<span>{children}</span>);
```

Comment:
Under what circumstances would this not be valid? This still wraps an invalid element in span tags.